<launch>
	<node pkg="ldtravocean" type="model_node" name="rov_node"
		args="$(env SIM_MODEL)" if="$(env IS_SIM)">
		<!-- This will also adjust the model sampling time -->
		<param name="Rate" value="10" />
		<param name="ModelWrap" value="10" />
		<remap from="tauIn" to="tauOut" />
		<remap from="meas" to="meas_sim" />
		<param name="maxThrust" value="140" />
		<param name="minThrust" value="-140" />
		<param name="thrustScale" value="100" />
		<param name="publish_world" value="true" />
		<param name="use_noise" value="true" unless="$(env NO_NOISE)" />

		<remap from="imu_model" to="imu" />
		<remap from="fix" to="gps" />
	</node>
</launch>
